content/logs/faq/setting-file-permissions-for-rotating-logs.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Setting file permissions for rotating logs (linux)
+kind: faq
+further_reading:
+- link: "/logs/faq/log-parsing-best-practice"
+  tag: "FAQ"
+  text: Log Parsing - Best Practice
+- link: "/logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "/logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
+---
+
+The datadog agent runs under the dd-agent user and dd-agent group. This prevents dd-agent accessing the logs in `/var/log` as they are only accessible by root (or a sudo admin).
+
+## Setting permissions using ACL's
+
+In order to allow read only access for dd-agent only, [create ACL's and modify logrotate to persist the permissions changes](https://help.ubuntu.com/community/FilePermissionsACLs).
+
+### Verifying ACL's are enabled on your system
```

Comment:
We should probably ensure that here and elsewhere it's ACLs not ACL's.

Reply:
👌